src/lsdb/loaders/dataframe/margin_catalog_generator.py

```diff
@@ -71,23 +71,34 @@ def _resolve_margin_order(self):
         if margin_pixel_mindist * 60.0 < self.margin_threshold:
             raise ValueError("margin pixels must be larger than margin_threshold")
 
-    def create_catalog(self) -> MarginCatalog | None:
+    def create_catalog(self) -> MarginCatalog:
         """Create a margin catalog for another pre-computed catalog
 
         Returns:
             Margin catalog object, or None if the margin is empty.
         """
+        if self.margin_order < 0 and self.margin_threshold <= 0:
+            return self.create_empty_margin_catalog()
```

Comment:
In this case we're actually not generating a margin catalog, so this should return None